tests/postgresql-tests.py

```diff
@@ -97,6 +98,14 @@ def main():
             if not args.unattended_debugging:
                 print("Press any key to launch tests.")
                 raw_input()
+        
+        # Get msg_handler from environment variable, default to 'notice'
+        msg_handler = os.environ.get('TDS_FDW_MSG_HANDLER', 'notice')
```

Comment:
Is this usually present as environment variable? It would usually be better to have all the stuff via parameters (same as before, this way it's automatically documented as part of the help).